packages/@tinacms/toolkit/src/packages/fields/plugins/GroupListFieldPlugin.tsx

```diff
@@ -83,29 +84,40 @@ const Group = ({ tinaForm, form, field, input }: GroupProps) => {
     [field.itemProps]
   )
 
+  // @ts-ignore
+  const isMax = items.length >= (field.max || Infinity)
+  // @ts-ignore
+  const isMin = items.length <= (field.min || 0)
```

Comment:
```suggestion
  const isMin = items.length < (field.min || 0)
```

Reply:
Same as what I said above for is max but in reverse. With `<` a list with 0 items would be valid when the min is set to 1.

Reply:
Here's that same field but with only one item, so the delete button is now disabled but you can add items again:
<img width="362" alt="Screenshot 2023-01-11 at 1 39 59 PM" src="https://user-images.githubusercontent.com/5075484/211878616-2667e38f-9c32-494b-9eee-809505289018.png">